test/1_stdlib/UnfoldSequence.swift

```diff
@@ -0,0 +1,48 @@
+// RUN: %target-run-simple-swift
```

Comment:
After giving it another minute's thought, I suppose this is fine. `StdlibCollectionUnittest` isn't exposed to normal tests, so it's probably safe to assume most of the sequence/collection tests are already validation tests, so I guess it wouldn't be unusual to put all of the `sequence(...)` tests into the validation test file.